.gitignore

```diff
@@ -194,3 +194,6 @@ FakesAssemblies/
 
 # Visual Studio 6 workspace options file
 *.opt
+
+# Ignore .idea/ Configuration files
+.idea/
```

Comment:
@oscarbatlle This is fine, but I would consider adding this to your global gitignore